celery/bin/worker.py

```diff
@@ -244,6 +252,8 @@ def get_options(self):
                    default=conf.CELERYD_MAX_TASKS_PER_CHILD, type='int'),
             Option('--prefetch-multiplier', dest='prefetch_multiplier',
                    default=conf.CELERYD_PREFETCH_MULTIPLIER, type='int'),
+            Option('--maxmemkb', dest='max_mem_kb',
```

Comment:
Maybe be consistent with `max_tasks_per_child` here, and have `max_mem_per_child`.  I inherited that naming from multiprocessing, but now that we use it's useful to be consistent.